mythril/laser/ethereum/instructions.py

```diff
@@ -1694,14 +1739,14 @@ def call_(self, global_state: GlobalState) -> List[GlobalState]:
                 global_state.new_bitvec("retval_" + str(instr["address"]), 256)
             )
             return [global_state]
-        global_state.mstate.stack.append(
-            global_state.new_bitvec("retval_" + str(instr["address"]), 256)
-        )
 
         native_result = native_call(
             global_state, callee_address, call_data, memory_out_offset, memory_out_size
         )
         if native_result:
+            global_state.mstate.stack.append(
```

Comment:
Don't we know the success/failure of the native call?

Reply:
There's currently a `# TODO: maybe use BitVec here constrained to 1` at the bottom of `native_call()`, which is probably a better place to put this anyway.